swagger.yaml

```
@@ -6503,6 +6503,7 @@ definitions:
       - 'no-skip-installation-disk'
       - 'no-skip-missing-disk'
       - 'no-ip-collisions-in-network'
+      - 'iscsi-host-network-interface-does-not-belong-to-machine-cidr'
```

let make it a bit shorter -- It still OK leaving it as is 
also better you have a validation be a `True` if it's good and `False` when it fails - just a bit easier to understand when seen/debugging 

@AlonaKaplan what to you think ?
```suggestion
      - 'iscsi-net-belong-to-machine-cidr'
```

---

maybe `iscsi-nic-belong-to-machine-cidr` WDYT?